### PR TITLE
Inline the `_fetchDocument` helper function in `getDocument`

### PR DESCRIPTION
**Pull request by Snuffleupagus**
This function has been modified a number of times over the years, and at this point it's small/simple enough that we can just inline the code instead.

### CODE REVIEW

File: src/display/api.js [hunk 1]
```diff
@@ -398,9 +398,16 @@ function getDocument(src) {
...
+      }
 
-      const workerIdPromise = _fetchDocument(worker, fetchDocParams);
-      const networkStreamPromise = new Promise(function (resolve) {
+      const workerIdPromise = worker.messageHandler.sendWithPromise(
+        "GetDocRequest",
+        docParams,
+        data ? [data.buffer] : null
+      );
+      const networkStreamPromise = new Promise(resolve => {
```
Comment (nicolo-ribaudo) [inline src/display/api.js:410]: `resolve` inside this Promise constructor is always called synchronously -- is it actually needed to have it as a promise?

Comment (Snuffleupagus) [reply src/display/api.js]: As with many other things that were once necessary, other changes have removed the need for this to be async :-)

**Review by timvandermeij — APPROVED** [review]

**Pull request merged**